spk/synocli-devel/Makefile

```diff
@@ -4,10 +4,10 @@ SPK_REV = 1
 SPK_ICON = src/gdb.png
 CHANGELOG = "1. Initial release of synocli-devel"
 
-#DEPENDS = cross/$(SPK_NAME)
+DEPENDS = cross/bc
 
```

Comment:
IMO `bc` can be removed as it is available with [SynoCli Misc.](https://github.com/SynoCommunity/spksrc/wiki/FAQ-SynoCliMisc)

Reply:
I was following your list from https://github.com/SynoCommunity/spksrc/issues/3253...